snips-nlu-ontology-parsers-ffi-macros/src/builtin_entity_parser.rs

```diff
@@ -101,7 +104,7 @@ pub fn extract_entity(
     };
     let opt_filters = opt_filters.as_ref().map(|vec| vec.as_slice());
 
-    Ok(parser.extract_entities(sentence, opt_filters))
+    Ok(parser.extract_entities(sentence, Some(reference_timestamp), opt_filters))
 }
```

Comment:
I think you should make `reference_timestamp` optional here through a `-1` value, not ideal but would let the rust code handle the default logic. The ffi may be used by other bindings than just the python one.

Reply:
Done, I went with a i64:MIN for default value, instead of -1, it is much less probable that someone wants to set reference time to 1901-12-13 20:45:52 than 1969-12-31 23:59:59.